### DISCUSSION TITLE
Add meta information (filename, lineNumbers) to code blocks

### DISCUSSION THREAD

**Discussion by yaldram** [mdx-js/mdx#2146] [Q&A] [answered]
Hello Community, I hope you all are doing great. I am using `@mdx-js/react` version - `2.1.3` with Gatsby version `4`, I am also using `prism-react-renderer` for syntax highlighting.

In my mdx files when I pass meta information for the code blocks - 
```
```js file=example.js
// fizzbuzz in JS
for (let i = 1; i <= 100; i++) {
  let out = "";
  if (i % 3 === 0) out += "Fizz";
  if (i % 5 === 0) out += "Buzz";
  console.log(out || i);
}
```

I am unable to access the `file` meta information as a prop in my component. Does this not work in version 2 . I was following this tutorial and there it is working as expected - https://annarossetti.com/articles/custom-code-blocks/

Please help I am stuck on this. Thanks for your time :)

**Comment by wooorm** [comment, marked as answer]
https://mdxjs.com/guides/syntax-highlighting/#syntax-highlighting-with-the-meta-field
